lib/common/BaseLoader.rb

```diff
@@ -10,12 +10,13 @@ module BaseLoader
       #
       # dir               - the directory to load resources from
       # individual_loader - the function that loads a resource from each file name
+      # json              - indicates if the resources are in json format
```

Comment:
ah good find. Turns out you can't have parameters after a block parameter or something...